### PR TITLE
Add tgrade

### PR DESCRIPTION
**Pull request by maurolacy**
Things pending / for revision:

- `updatelink`. (not exactly sure what that is. Needs schema in main tgrade repo).
- ~~`key_algos`. (is this correct (I think so) / complete)~~ (done)
- ~~`slip44`. (don't know what that is)~~ (use most common value: 118)
- ~~low/average/high gas prices. (confirm / adjust values)~~ (done)
- `binaries`. (do we add a binary release to tgrade?)
- ~~seed / persistent peers **names**.~~
- ~~`grpc` endpoint. (do we have a public one?)~~ (private. removed)
- ~~logo image. (asked Alexia for it)~~ (done)
- ~~tgd token images. (idem)~~ (done)

### CODE REVIEW

File: tgrade/chain.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "utgd",
+                "fixed_min_gas_price": 0.05,
+                "low_gas_price": 0,
+                "average_gas_price": 0.075,
+                "high_gas_price": 0.1
```
Comment (abefernan) [inline tgrade/chain.json:26]: The app needs to config low/avg/high for Keplr and we're using:

- avg: 0.05
- low: avg / 2 = 0.025
- high: avg * 2 = 0.1

Not sure about fixed_min_gas_price

File: tgrade/chain.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    "bech32_prefix": "tgrade",
+    "daemon_name": "tgrade",
+    "node_home": "$HOME/.tgrade",
+    "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/confio/tgrade-networks/main/mainnet-1/config/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
```
Comment (abefernan) [inline tgrade/chain.json:18]: Seems like yet another repo in which to register TGD? https://github.com/futpib/slip44/blob/master/src/slip44.ts

Comment (abefernan) [reply tgrade/chain.json:18]: As you can see here each coin has an ID: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Comment (maurolacy) [reply tgrade/chain.json:18]: OK. Let's remove this by now then.

File: tgrade/chain.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "tgrade",
+    "status": "live",
+    "network_type": "mainnet",
+    "updatelink": "https://raw.githubusercontent.com/confio/tgrade/main/chain.schema.json",
+    "pretty_name": "Tgrade",
+    "chain_id": "mainnet-1",
```
Comment (abefernan) [inline tgrade/chain.json:8]: According to [the endpoint](https://rpc.mainnet-1.tgrade.confio.run/status) shouldn't chain-id be `tgrade-mainnet-1`?

File: tgrade/chain.json (hunk 2 above)
Comment (maurolacy) [reply tgrade/chain.json:18]: Seems like just using "118" here is a safe bet.

File: tgrade/chain.json (hunk 3 above)
Comment (maurolacy) [reply tgrade/chain.json:8]: You are right! Good catch.

**Review by HannuProd — APPROVED** [review]

**Pull request merged**